### PR TITLE
Update nuspecs

### PR DESCRIPTION
**Pull request by rainersigwald**

### CODE REVIEW

**Comment by AndyGerlicher** [comment]
LGTM

File: src/.nuget/packageLoad.targets [hunk 1]
```diff
@@ -38,31 +38,17 @@
   <Target Name="_RestoreBuildToolsPackagesConfig"
```
Comment (cdmihai) [inline src/.nuget/packageLoad.targets]: Should this target have a dual xplat / master behaviour? On xplat it uses CLI and project.json, on master it uses packages.config. Or should we just pick the CLI from buildtools for both?

**Comment by cdmihai** [comment]
Don't know if this worth doing now but: As long as we still have that nuspec reference generator still working, it might be worth having it delete unused nuspec references, if there are any. To do that, you delete all references except the p2p msbuild ones, and run the build. The reference generator should add back only the stuff that's required.

File: src/.nuget/packageLoad.targets (hunk 1 above)
Comment (rainersigwald) [inline src/.nuget/packageLoad.targets]: In the glorious future, we'll just do "dotnet restore". For now, I'm just letting this file diverge between xplat and master.

**Comment by rainersigwald** [comment]
@cdmihai Ok, I'll do that instead of this second commit. Just a sec.

**Comment by rainersigwald** [comment]
@cdmihai Totally worth doing now since it caught two stale references that still pointed to non-RTM packages. Good catch!

**Comment by cdmihai** [comment]
:shipit::

**Pull request merged**